core/themes/basis/template.php

```diff
@@ -38,6 +38,20 @@ function basis_preprocess_page(&$variables) {
     $variables['classes'][] = 'view-name-' . $view->name;
   }
 
+  // Get the installed version.
+  $update_version = config_get('basis.settings', 'css_update_version');
+  $update_preference = config_get('basis.settings', 'css_update');
+
+  // Process supplemental CSS versions.
+  $supplemental_css_versions = basis_supplemental_css_versions();
+  foreach ($supplemental_css_versions as $supplemental_css_version) {
+    if ($update_preference === 'all_updates' || version_compare($update_version, $supplemental_css_version, '>=')) {
+      $class_version = preg_replace('/^([0-9.]+)\.0$/', '${1}', $supplemental_css_version);
+      $supplemental_css_version_class = 'update-' . str_replace('.', '-', $class_version);
```

Comment:
This regex looks like it strips off the trailing `.0` if one is present. i.e. it makes `1.30.0` become `1.30`. However, it doesn't have an effect on any version that ends in a non-zero. i.e. `1.30.1` stays the same and would result in the class `update-1-30-1`.

I think supporting bug fix versions is a good idea for maximum flexibility (even if we don't use them). Should we keep stripping off the trailing `.0` on minor versions?

Reply:
Actually now that `basis_supplemental_css_versions()` _already_ shortens the versions to `1.30`, we don't need this at all. It will just use whatever version in in that function.

Reply:
The regex has been removed now that we no longer need it.